### PR TITLE
Add recomendation to install operator before alertmanager/prometheus

### PR DESCRIPTION
**Pull request by gianrubio**
Because the `thirdypartyresource` is not yet installed, you got an error when you are trying to install  prometheus/alermanager.

```
$ helm install alertmanager --name alertmanager --debug
[debug] Created tunnel using local port: '57662'

[debug] SERVER: "localhost:57662"

[debug] CHART PATH: /Users/grubio/work/src/github.com/coreos/prometheus-operator/helm/alertmanager

Error: apiVersion "monitoring.coreos.com/v1alpha1" in alertmanager/templates/alertmanager.yaml is not available
```

### CODE REVIEW

**Comment by brancz** [comment]
lgtm, I'll leave the final call up to @mgoodness .

Wouldn't it be great if helm had the ability to specify dependencies?

File: helm/alertmanager/README.md [hunk 1]
```diff
@@ -4,6 +4,8 @@ Installs a [Prometheus](https://prometheus.io) Alertmanager instance using the C
 
 ## TL;DR;
 
+Before install alertmanager you should install [prometheus operator](https://github.com/coreos/prometheus-operator/blob/master/helm/prometheus-operator/README.md).
```
Comment (brancz) [inline helm/alertmanager/README.md]: Before install -> Before installing

File: helm/prometheus/README.md [hunk 2]
```diff
@@ -4,6 +4,8 @@ Installs a [Prometheus](https://prometheus.io) instance using the CoreOS [promet
 
 ## TL;DR;
 
+Before install prometheus you should install [prometheus operator](https://github.com/coreos/prometheus-operator/blob/master/helm/prometheus-operator/README.md).
```
Comment (brancz) [inline helm/prometheus/README.md]: Before install -> Before installing

**Comment by gianrubio** [comment]
> Wouldn't it be great if helm had the ability to specify dependencies?

I don't know what the goals of this charts, I would like to see this charts on [kubernetes/charts](http://github.com/kubernetes/charts). In that case makes no sense to specify prometheus and alert manager as a dependency of operator.

**Comment by gianrubio** [comment]
I just realise that these charts are not deploying alert manager and prometheus. So it's nice to have this charts as a dependencies.

**Comment by gianrubio** [comment]
Is prometheus-operator a dependencie of alertmanager/prometheus or the opposite? 

Make sense to have prometheus operator without alertmanager/prometheus?

**Comment by mgoodness** [comment]
>Wouldn't it be great if helm had the ability to specify dependencies?

Helm supports dependencies, and my original `prometheus` and `alertmanager` charts included the `prometheus-operator` one. Unfortunately, Helm (currently) has no way to dictate the order of installation, so deployments were unreliable. So now `prometheus-operator` has to be installed first. Just failed to document that fact.

>I would like to see this charts on kubernetes/charts

As one of the maintainers of that repo, I can say we're in favor of upstream projects (like this one) creating & maintaining their own charts. It's easier to keep a chart updated with the latest application functionality when it lives alongside the code. There's also an existing `prometheus` chart that doesn't use the operator.

>Is prometheus-operator a dependencie of alertmanager/prometheus or the opposite?

prometheus-operator must be installed (by the chart or some other method) before the `alertmanager` or `prometheus` charts can be deployed.

>Make sense to have prometheus operator without alertmanager/prometheus?

Yes. We at Ticketmaster will use this `prometheus-operator` chart, but maintain our own `alertmanager` and `prometheus` charts because of customization.

File: helm/alertmanager/README.md [hunk 3]
```diff
@@ -4,6 +4,8 @@ Installs a [Prometheus](https://prometheus.io) Alertmanager instance using the C
 
 ## TL;DR;
 
+Before installing alertmanager you should install [prometheus operator](https://github.com/coreos/prometheus-operator/blob/master/helm/prometheus-operator/README.md).
+
```
Comment (mgoodness) [inline helm/alertmanager/README.md]: There is a `Prerequisites` section in the README. Just add the `prometheus-operator` chart link as a second bullet.

File: helm/prometheus/README.md [hunk 4]
```diff
@@ -4,6 +4,8 @@ Installs a [Prometheus](https://prometheus.io) instance using the CoreOS [promet
 
 ## TL;DR;
 
+Before installing prometheus you should install [prometheus operator](https://github.com/coreos/prometheus-operator/blob/master/helm/prometheus-operator/README.md).
+
```
Comment (mgoodness) [inline helm/prometheus/README.md]: See above comment.

File: helm/alertmanager/README.md (hunk 3 above)
Comment (gianrubio) [inline helm/alertmanager/README.md]: done
